digdag-core/src/main/java/io/digdag/core/database/DatabaseProjectStoreManager.java

```diff
@@ -292,10 +290,17 @@ public StoredWorkflowDefinitionWithProject getLatestWorkflowDefinitionByName(int
         public List<StoredWorkflowDefinitionWithProject> getLatestActiveWorkflowDefinitions(
                 int pageSize,
                 Optional<Long> lastId,
+                Optional<String> name,
                 AccessController.ListFilter acFilter)
             throws ResourceNotFoundException
         {
-            return autoCommit((handle, dao) -> dao.getLatestActiveWorkflowDefinitions(siteId, pageSize, lastId.or(0L), acFilter.getSql()));
+            return autoCommit((handle, dao) -> dao.getLatestActiveWorkflowDefinitions(
+                    siteId,
+                    pageSize,
+                    lastId.or(0L),
+                    name.isPresent() && !name.get().isEmpty() ? "%" + escapeLikeParameter(name.get()) + "%" : "%",
```

Comment:
If `name` parameter is `absent`, this method just sets `%` so that the name filter does not affect.